### PR TITLE
via: allow vkvia to find SDK in 1.1.108

### PR DESCRIPTION
**Pull request by bradgrantham-lunarg**
add loop over SDK explicit layer path dirs in PrintSystemSdkInfo

add 1.1.108 explicit layer path dir so vkvia succeeds on 1.1.108

### CODE REVIEW

File: via/via_system_linux.cpp [hunk 1]
```diff
@@ -1139,23 +1139,28 @@ ViaSystem::ViaResults ViaSystemLinux::PrintSystemSdkInfo() {
...
+		    if (NULL != strstr(cur_ent->d_name, vulkan_so_prefix) && strlen(cur_ent->d_name) == 14) {
+		    }
+		}
+		closedir(sdk_dir);
+
+		result = PrintExplicitLayersInFolder("", explicit_layer_path);
+
+		_found_sdk = true;
+		_sdk_path = sdk_path;
+		sdk_exists = true;
```
Comment (MarkY-LunarG) [inline via/via_system_linux.cpp]: Should we break here?

Comment (bradgrantham-lunarg) [reply via/via_system_linux.cpp]: We could.  Then it would preferentially find earlier SDKs.

Comment (bradgrantham-lunarg) [reply via/via_system_linux.cpp]: Sure!  I guess we won't have both explicit layer paths in one VULKAN_SDK.

**Comment by bradgrantham-lunarg** [comment]
Abandoned

**Pull request closed without merging**